x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -157,109 +158,96 @@ export class HeadlessChromiumDriverFactory {
           .subscribe(observer)
       );
 
-      // Register with a few useful puppeteer event handlers:
-      // https://pptr.dev/#?product=Puppeteer&version=v1.10.0&show=api-event-error
-      // https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#class-page
-
-      const stderr$ = Rx.fromEvent(page as NodeJS.EventEmitter, 'console').pipe(
-        filter((line: any) => line._type === 'error'),
-        map((line: any) => line._text),
-        share()
-      );
-
-      const [consoleMessage$, message$] = partition(
-        (msg: string) => !!msg.match(/\[\d+\/\d+.\d+:\w+:CONSOLE\(\d+\)\]/)
-      )(stderr$);
```

Comment:
Removing this code as it does not work with the puppeteer driver. I believe this was written at a time when we launched Chromium directly, without a helper. As of now, we no longer have a way to scan the browser's stdin / stderr as a stream.

Reply:
I believe this only works if you use `dumpio` in puppeteer launch: https://pptr.dev/#?product=Puppeteer&version=v1.19.0&show=api-puppeteerlaunchoptions. We don't expose an option to do this, so I'm cool with removing this

Reply:
Same here. As we have explored using `dumpio` before and found that it doesn't provide a stream of helpful info for us.